src/Pickles/Pickles.TestFrameworks.UnitTests/NUnit/NUnit2/WhenDeterminingTheSignatureOfAnNUnitExampleRow.cs

```diff
@@ -48,5 +48,19 @@ public void ThenCanSuccessfullyMatch()
             var isMatch = signature.IsMatch("Pickles.TestHarness.AdditionFeature.AddingSeveralNumbers(\"40\",\"50\",\"90\",System.String[])".ToLowerInvariant());
             Check.That(isMatch).IsTrue();
         }
+
+		[Test]
+	    public void ThenCanSuccesfullyMatchSpecialCharacters()
```

Comment:
"Successfully" is written with a double ss before the f :-)

Also please replace the spaces with tabs.